scripts/maint/practracker/exceptions.txt

```diff
@@ -56,7 +56,7 @@ problem dependency-violation /src/core/crypto/onion_crypto.c 5
 problem dependency-violation /src/core/crypto/onion_fast.c 1
 problem dependency-violation /src/core/crypto/onion_tap.c 3
 problem dependency-violation /src/core/crypto/relay_crypto.c 9
-problem file-size /src/core/mainloop/connection.c 5569
+problem file-size /src/core/mainloop/connection.c 5694
```

Comment:
Wow, you shouldn't need to fix practracker on 0.4.2 or 0.4.3.

I have removed check-best-practices from check-local on all maint and release branches. So practracker only runs by default on master.